code/modules/mob/living/carbon/emote.dm

```diff
@@ -220,6 +223,7 @@
 	key = "chuckle"
 	key_third_person = "chuckles"
 	message = "chuckles."
+	message_mime = "puts a hand on their chest and imitates a smug chuckle."
```

Comment:
i have no hands

Reply:
I do

Reply:
You can't use that excuse anymore.

Reply:
no really, you can't assume the mime's hands haven't been removed

Reply:
I know, thanks for pointing it out (you arm-having freak)